README.md

```diff
@@ -39,30 +40,24 @@ It should look like this
 ```ruby
 RSpec.configure do |config|
   # Existing code
+  config.after(:each, type: :controller) do
+    SmashingDocs.run!(request, response, true)
+  end
   config.after(:suite) { SmashingDocs.finish! }
 end
 ```
-#### To run on all controller tests
 
-Add this to your `spec_helper.rb`
+#### To run on only select tests
+Set the `c.run_all` line to `false` in `rails_helper.rb`
 ```ruby
-config.after(:each, type: :controller) do
-  SmashingDocs.run!(request, response)
+SmashingDocs.config do |c|
+  c.template_file = 'smashing_docs/template.md'
+  c.output_file   = 'smashing_docs/api_docs.md'
+  c.run_all       = false
 end
 ```
 
-The whole file should look like this
-```ruby
-RSpec.configure do |config|
-  # Existing code
-  config.after(:each, type: :controller) do
-    SmashingDocs.run!(request, response)
-  end
-  config.after(:suite) { SmashingDocs.finish! }
-end
-```
-#### To run on only select tests
-Just add `SmashingDocs.run!(request, response)` to specific tests
+Then just add `SmashingDocs.run!(request, response)` to the tests you want to run
```

Comment:
This is not for this phase, so don't change anything here and maybe just make a card about it so we don't forget. But I think eventually it would be nice see this and the information aliased into a cleaner method for people who are only using the explicit option. Maybe something like:
`SmashingDocs.include` which would tell it to use that test without any additional information, and `SmashingDocs.include(information: {hash here}` would include it with information. It might cause issues since it isn't passing the request and response directly, but who knows. This is in response to seeing some places where there's both a `SmashingDocs.run!(request, response)` and a `SmashingDocs.information(...)` in the same test, and thinking there might a cleaner way.